.changeset/six-flowers-smash.md

```diff
@@ -0,0 +1,15 @@
+---
+"@bigtest/agent": minor
+"@bigtest/atom": minor
+"bigtest": minor
```

Comment:
Let's make "@bigtest/atom" a minor, and "bigtest" a patch. Everything else will get a patch because it depends on atom and it got a minor bump.